docs/resources/index.md

```diff
@@ -0,0 +1,188 @@
+# Resources
+
```

Comment:
and this?

Reply:
I have added an autogeneration in the next commit.